flask/app/models.py

```diff
@@ -421,25 +424,36 @@ class Variation(str, Enum):
     Stop_Lost = "Stop_Lost"
```

Comment:
Good catch! I'd say it's pretty redundant but they were taken verbatim from the reports. I can remove `_Variant` from both the enum names and the values.

Reply:
If they're verbatim from the reports, I assume they're fairly static unless we have a pipeline update that detects more types of variants, but a string field may also be valid to avoid a database migration when that happens. If `_Variant` is verbatim then it's easier to keep it instead of having to remap.